GOVERNANCE.md

```diff
@@ -705,21 +705,21 @@ provide examples on how to assign permissions:
     of the pymc-devs organization
 
 - Carme, core contributor, doc team
-  * Added to the Discourse PyMC_team and given "leader" trust level
+  * Added to the community PyMC_team and given "leader" trust level
   * Added to all private communication channels
   * Write permissions on pymc-examples and resources repositories, triage permissions
     to pymc and pymc-experimental repositories
-  * Admin access to ReadTheDocs accounts
+  * Administrative access to ReadTheDocs accounts
 
-- Dolors, core contributor, dev and discourse teams
-  * Added to the Discourse PyMC_team and given "leader" trust level,
-    given moderator permissions on Discourse (that might be rotating and not always active)
+- Dolors, core contributor, dev and community teams
+  * Added to the Community PyMC_team and given "leader" trust level,
+    given moderator permissions on Discourse (such permissions may rotate and not always be active)
   * Added to all private communication channels
-  * Write permissions on pymc and pymc-experimental repositories, triage permissions
-    to pymc-examples and resources repositories
+  * Write permissions on pymc, pymc-experimental, pymcon, pymc-data-umbrella, and other "event" repositories, triage permissions
+    on pymc-examples and resources repositories
 
 - Eudald, core contributor, no dedicated team membership
-  * Added to the Discourse PyMC_team and given "leader" trust level
+  * Added to the Community PyMC_team and given "leader" trust level
   * Added to all private communication channels
   * Triage permissions on all repositories
   * Access to pymc_devs twitter account as they are the main manager
```

Comment:
```suggestion
  * Access to pymc gmail, drive and grant application platforms (i.e. CZI slack) as they are the main grant writing coordinator
```
Updated that so we keep a no-team member here with special and important permissions.